components/centos/centos-mesos-marathon-singlenode-setup/provisioning/playbook.yml

```diff
@@ -0,0 +1,40 @@
+- hosts: localhost
+  sudo: yes
+  connection: local
+  tasks:
+          - name: Install Mesosphere repository
+            yum: name=http://repos.mesosphere.com/el/7/noarch/RPMS/mesosphere-el-repo-7-3.noarch.rpm state=present
+
+          - name: Installing zookeeper, mesos, marathon, docker, git
+            yum: name={{item}} state=latest
+            with_items:
+                - mesosphere-zookeeper 
+                - mesos 
+                - marathon
+                - docker
+                - git
+                - python-pip
+            register: result
+
+          - name: Starting zookeeper, mesos, marathon, docker
+            service: name={{item}} state=started enabled=yes
+            with_items:
+                - zookeeper
+                - mesos-master
+                - mesos-slave
+                - marathon
+                - docker
+
+          - name: Clone atomicapp
```

Comment:
I think I added it so that the box can readily be used to run `atomicapp`. Looking at it again, I feel it's not required here in the playbook. Removing it.